src/dotnet/commands/dotnet-add/dotnet-add-package/LocalizableStrings.cs

```diff
@@ -9,21 +9,21 @@ internal class LocalizableStrings
 
         public const string AppDescription = "Command to add package reference";
 
-        public const string CmdPackageDescription = "Package reference to add";
+        public const string CmdPackageDescription = "The package reference to add.";
 
-        public const string SpecifyExactlyOnePackageReference = "Please specify one package reference to add.";
+        public const string SpecifyExactlyOnePackageReference = "Please specify only one package reference to be added.";
 
-        public const string CmdFrameworkDescription = "Add reference only when targeting a specific framework";
+        public const string CmdFrameworkDescription = "Adds reference only when targeting a specific framework.";
 
-        public const string CmdNoRestoreDescription = "Add reference without performing restore preview and compatibility check.";
+        public const string CmdNoRestoreDescription = "Adds reference without performing restore preview and compatibility check.";
 
-        public const string CmdSourceDescription = "Use specific NuGet package sources to use during the restore.";
+        public const string CmdSourceDescription = "Uses specific NuGet package sources to use during the restore.";
```

Comment:
True @livarcocc. How about "Specifies a NuGet package source to use during the restore."? 
It's only one source, right?